tests/integration/targets/vmware_cluster_vcls/tasks/main.yml

```diff
@@ -0,0 +1,117 @@
+---
+- name: Test On Simulator
+  when: run_on_simulator
+  block:
+    - name: Set vCLS Settings In Cluster
+      vmware.vmware.cluster_vcls:
+        validate_certs: false
+        hostname: "{{ vcenter_hostname }}"
+        username: "{{ vcenter_username }}"
+        password: "{{ vcenter_password }}"
+        datacenter: "{{ vcenter_datacenter }}"
+        cluster: "{{ test_cluster }}"
+        port: "{{ vcenter_port }}"
+        allowed_datastores: "{{ allowed_datastores }}"
+    # The simulator never seems to update its vCLS settings, so theres nothing to validate here
+
+- name: Test On VCenter
+  when: not run_on_simulator
+  block:
+    - name: Import common vars
+      ansible.builtin.include_vars:
+        file: ../group_vars.yml
+    - name: Create Test Cluster
+      community.vmware.vmware_cluster:
```

Comment:
wouldn't it be better to use `vmware.vmware.cluster` module instead now that it's merged? 

same for the task to destroy the test cluster